src/modules/Elsa.Workflows.Core/Contexts/WorkflowExecutionContext.cs

```diff
@@ -572,9 +579,9 @@ internal void TransitionTo(WorkflowSubStatus subStatus)
     /// <summary>
     /// Creates a new <see cref="ActivityExecutionContext"/> for the specified activity.
     /// </summary>
-    public ActivityExecutionContext CreateActivityExecutionContext(IActivity activity, ActivityInvocationOptions? options = default)
+    public async Task<ActivityExecutionContext> CreateActivityExecutionContext(IActivity activity, ActivityInvocationOptions? options = default)
```

Comment:
Please mind the naming convention to apply the `Async` suffix when changing methods into async.